backend/resources.py

```diff
@@ -8,3 +8,4 @@ class ResourceName(Enum):
     DATASET_VERSIONS_ENDPOINT_FUNCTION_NAME = f"{ENV}-dataset-versions"
     IMPORT_STATUS_ENDPOINT_FUNCTION_NAME = f"{ENV}-import-status"
     USERS_ROLE_NAME = f"{ENV}-data-lake-users"
+    LINZ_READ_ONLY_ROLE_NAME = f"{ENV}-data-lake-linz-readonly"
```

Comment:
Would removing the "LINZ" part of the name make sense? We might want to grant access to it for non-LINZ users in the future.